src/InsertCalorimeter_geo.cpp

```diff
@@ -89,7 +89,7 @@ static Ref_t createDetector(Detector& desc, xml_h handle, SensitiveDetector sens
 
   // Assembly that will contain all the layers
   Assembly assembly(detName);
-
+  assembly.setVisAttributes(desc.visAttributes("InvisibleWithDaughters"));
```

Comment:
This plugin is used for ecal insert too, I think. Making it invisible in all cases will cause confusion about whether it is enabled or not.
```suggestion

```

Reply:
Otherwise the visualization of the insert is broken (regardless of which insert) we probably need it on

Reply:
```suggestion
  // Workaround for https://github.com/eic/epic/issues/411
  assembly.setVisAttributes(desc.visAttributes("InvisibleWithDaughters"));
```